app/code/Magento/Checkout/view/frontend/web/js/view/minicart.js

```diff
@@ -120,7 +120,9 @@ define([
          * Close mini shopping cart.
          */
         closeMinicart: function () {
-            $('[data-block="minicart"]').find('[data-role="dropdownDialog"]').dropdownDialog('close');
+            try {
+                $('[data-block="minicart"]').find('[data-role="dropdownDialog"]').dropdownDialog('close');
+            } catch (err) {}
```

Comment:
Could you please explain why you added here a empty catch block?

Reply:
I simply ignore it.
When page loads,  dropdownDialog component does not execute so the element [data-role="dropdownDialog"] call dropdownDialog will return error.

Reply:
with new commit for trigger binding scope only  with element display in viewport, this change not need anymore